### PR TITLE
Fixed bugs according to last night's run

### PR DESCRIPTION
**Pull request by max-landry**
There were a number of small things that I caught and fixed that are not present in the master branch

### CODE REVIEW

**Comment by l-p-d** [comment]
Most of those things are fixed in master, and that branch is many commits behind. Could you create a new branch from master and pull request from that?

**Comment by xeoth** [comment]
Is this PR finished and merge-able or not yet?

**Comment by max-landry** [comment]
i believe so, though there are more bugs to be fixed due to what was mentioned in discord

File: src/bot.py [hunk 1]
```diff
@@ -110,7 +114,7 @@ def int_conv(string: str) -> bool:
         # creates database entry for commenter if required
         if db.get(commenter) == None:
             print("creating commenter db")
-            db.set_available(commenter, 3)
+            db.set_available(commenter, 10000)
```
Comment (xeoth) [inline src/bot.py:117]: 10 000? Why so much?

**Review by xeoth — APPROVED** [review]

**Pull request merged**